course/src/main/java/in/testpress/course/fragments/ExamStartScreenFragment.kt

```diff
@@ -120,4 +127,31 @@ class ExamStartScreenFragment : BaseExamWidgetFragment() {
             }
         }
     }
+
+    private fun showExamStatusViews(exam: DomainExamContent) {
+        if (exam.isEnded()){
+            displayExamEndedView()
+        }
+        else {
+            displayAllTheBestView()
+        }
+    }
+
+    @SuppressLint("UseCompatLoadingForDrawables")
+    private fun displayExamEndedView(){
+        examStatusDescription.text = getString(R.string.testpress_exam_ended)
+        examStatusDescription.setTextColor(resources.getColor(R.color.testpress_red))
+
+        examStatusIcon.setImageDrawable(resources.getDrawable(R.drawable.ic_baseline_info_24))
```

Comment:
Since for both case you are using same icon, you don't need to specify it multiple times. You can just mention it once in xml.

Reply:
Okay @Karthik-0, Acually I missed something. I've planned to use different icons for these cases and I just missed that. I'll update the code.